Crash due to driver cap - Kubuntu 26.04

```
$ roc-send -vv -s rtp://192.168.1.100:10001
22:04:45.121 [5621] [dbg] roc_sndio: [sndfile_backend.cpp:20] sndfile backend: initializing
22:04:45.122 [5621] [err] roc_sndio: [array.h:279] array: can't allocate memory: current_cap=256 requested_cap=512 embedded_cap=256

src/internal_modules/roc_sndio/target_sox/roc_sndio/sox_backend.cpp:220: error: roc_panic()

ERROR: roc_sndio: sox backend: can't add driver

Aborted                    (core dumped) roc-send -vv -s rtp://192.168.1.100:10001
```

**Workaround:** I uninstalled `libsox-fmt-alsa` and roc-send started working again

This is on Kubuntu 26.04 with KDE
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 26.04 LTS
Release:        26.04
Codename:       resolute
```

```
$ apt-cache show roc-toolkit-tools
Package: roc-toolkit-tools
Source: roc-toolkit
Priority: optional
Section: universe/sound
Installed-Size: 2981
Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Architecture: amd64
Version: 0.4.0+dfsg-5ubuntu3
Depends: libc6 (>= 2.38), libpulse0 (>= 0.99.1), libsndfile1 (>= 1.0.20), libsox-ng3 (>= 14.7.0.3), libspeexdsp1 (>= 1.2.1), libstdc++6 (>= 4.1.1), libuv1t64 (>= 1.4.2)
Filename: pool/universe/r/roc-toolkit/roc-toolkit-tools_0.4.0+dfsg-5ubuntu3_amd64.deb
Size: 532662
MD5sum: 8f48fd3a2a2cd4632b5f787813f490df
SHA1: 12f8adaeeffc30968dfb7faa9f5550a624942d06
SHA256: 501ed3f14660129e7724bccc528f422d5d0272b9f025d68a15cd5d35f018b6a5
SHA512: 404d7eff90535f1fcc90a9c4cf2976b80f4054edaba54025914e81f98b83ec591414b6b8c9b5938b93cf065267fce905eea855e75b84bef56c9e0526fea34ccc
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -37,6 +37,9 @@ jobs:
       fail-fast: false
       matrix:
         include:
+          - script: linux-x86_64/ubuntu-26.04
+            image: rocstreaming/env-ubuntu:26.04
+
           - script: linux-x86_64/ubuntu-24.04
             image: rocstreaming/env-ubuntu:24.04
 

--- a/scripts/ci_checks/linux-x86_64/ubuntu-26.04.sh
+++ b/scripts/ci_checks/linux-x86_64/ubuntu-26.04.sh
@@ -1,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+for comp in gcc-15 clang-21
+do
+    scons -Q \
+          --enable-werror \
+          --enable-tests \
+          --enable-benchmarks \
+          --enable-examples \
+          --build-3rdparty=openfec \
+          --compiler=${comp} \
+          test
+done

--- a/src/internal_modules/roc_sndio/driver.h
+++ b/src/internal_modules/roc_sndio/driver.h
@@ -21,7 +21,7 @@ namespace sndio {
 class IBackend;
 
 //! Maximum number of drivers.
-static const size_t MaxDrivers = 256;
+static const size_t MaxDrivers = 512;
 
 //! Driver type.
 enum DriverType {

